packages/selecto/src/SelectoManager.tsx

```diff
@@ -67,34 +69,35 @@ import { PROPERTIES, injector, CLASS_NAME } from "./consts";
     };
     const getter = camelize(`get ${property}`);
     if (prototype[getter]) {
-        attributes.get = function get() {
+        attributes.get = function() {
```

Comment:
What do you think about removing the function name since it will be ignored when assigned as a method?